packages/angular/cli/src/commands/analytics/info/cli.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { tags } from '@angular-devkit/core';
+import { Argv } from 'yargs';
+import { analyticsConfigValueToHumanFormat } from '../../../analytics/analytics';
+import {
+  CommandModule,
+  CommandModuleImplementation,
+  Options,
+} from '../../../command-builder/command-module';
+import { getWorkspaceRaw } from '../../../utilities/config';
+
+export class AnalyticsInfoCommandModule
+  extends CommandModule
+  implements CommandModuleImplementation
+{
+  command = 'info';
+  describe = 'Prints analytics gathering and reporting configuration in the console.';
+  longDescriptionPath?: string | undefined;
+
+  builder(localYargs: Argv): Argv {
+    return localYargs.strict();
+  }
+
+  async run(_options: Options<{}>): Promise<void> {
+    const [globalWorkspace] = getWorkspaceRaw('global');
+    const [localWorkspace] = getWorkspaceRaw('local');
+    const globalSetting = globalWorkspace?.get(['cli', 'analytics']);
+    const localSetting = localWorkspace?.get(['cli', 'analytics']);
+
+    this.context.logger.info(tags.stripIndents`
+      Global setting: ${analyticsConfigValueToHumanFormat(globalSetting)}
+      Local setting: ${
+        this.context.workspace
+          ? analyticsConfigValueToHumanFormat(localSetting)
+          : 'No local workspace configuration file.'
+      }
+      Effective setting: ${analyticsConfigValueToHumanFormat(
```

Comment:
I imagine some users might not understand what "effective" means in this context? "Local" is also a bit overloaded as "project" in other situations. We should define these terms in `ng analytics info` help content and then maybe when we print the status, we can include a link to the AIO page for `ng analytics` which would include that content.

Reply:
I agree, although I'd prefer to leave this for a seperate PR and add the link then.